reworked docs and made component standards easier to follow/reference (#80)
AdaloHQ/docs: website/docs/component-standards/component-listing/logo.md

---
id: logo
title: Logo
---

> The logo in the marketplace is a chance for you to show off your component in a branded way. So while the thumbnail is meant to be a representation of what will actually show up when it’s in the app, the logo is a chance for you to have a little more fun and it’s more of an icon.

Reviewer: ```suggestion
The logo in the marketplace is a chance for you to show off your component in a branded way. So while the thumbnail is meant to be a representation of what will actually show up in the app, the logo is more of an icon. And it's a chance for you to have a little more fun!
```
